bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tf1.py

```diff
@@ -59,6 +59,22 @@ def pyf_tf1_coulomb(x, p):
 res = rtf1_coulomb.EvalPar(x[:, ::2], params)
 \endcode
 
+The below example defines a TF1 instance using the ROOT constructor, and sets its parameters using the Pythonized TF1::SetParameters function (i.e. without evaluating).
```

Comment:
```suggestion
The below example defines a TF1 instance using the ROOT constructor, and sets its parameters using the Pythonized TF1.SetParameters function (i.e. without evaluating).
```

Reply:
Thanks for the catch, I also changed a ``TF1::EvalPar`` to TF1.EvalPar a bit earlier in the docstring as well to use this syntax if that's OK.

Reply:
Yes that's good, thanks!